docs/source/tutorials_source/package/tutorial_timm_backbone.py

```diff
@@ -0,0 +1,91 @@
+"""
+.. _lightly-timm-backbone-tutorial-8:
+
+Tutorial 8: Using timm models as backbones
+===========================================
+
+You can use LightlySSL to pre-train any model using self-supervised learning since they
+share a similar workflow. All methods have a base model (the backbone), which can be any
```

Comment:
For me it is not clear with respect to what the `share a similar workflow` is meant.